packages/preprocessor/src/parsers/playwrightParser.ts

```
@@ -0,0 +1,192 @@
+ * Extrai e limpa mensagem de erro do objeto error do Playwright
+ * Remove códigos ANSI das mensagens
+ * @param error - Erro do Playwright
+ * @returns Mensagem de erro limpa ou null se não houver erro
+ */
+function extractErrorMessage(error: PlaywrightError | undefined): string | null {
+  if (!error) {
+    return null;
+  }
+
+  if (error.message) {
+    return error.message.replace(/\u001b\[[0-9;]*m/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)